tsl/test/sql/cagg_usage.sql.in

```diff
@@ -301,12 +301,11 @@ SELECT * FROM cagg1;
 CREATE MATERIALIZED VIEW cagg2 WITH (timescaledb.continuous,timescaledb.materialized_only=true) AS SELECT time_bucket('1 month', time, 'PST8PDT') FROM metrics GROUP BY 1;
 SELECT * FROM cagg2;
 
--- custom origin
+-- custom origin - not supported due to variable size
```

Comment:
This CAgg definition with a custom origin was permitted until now for unclear reasons.